protocol/x/clob/memclob/memclob.go

```diff
@@ -839,6 +849,8 @@ func (m *MemClobPriceTimePriority) ReplayOperations(
 	shortTermOrderTxBytes map[types.OrderHash][]byte,
 	existingOffchainUpdates *types.OffchainUpdates,
 ) *types.OffchainUpdates {
+	lib.AssertCheckTxMode(ctx)
```

Comment:
Suggestion, up to you: note this function should only be called in `PrepareCheckState`.